specification.json

```diff
@@ -320,6 +341,20 @@
             "RFC 2119 keyword": "MAY",
             "children": []
         },
+        {
```

Comment:
Depending on what we mean by breaking, new methods on an interface may be breaking depending on the language. (For instance the C# SDK uses C# 7.3, which doesn't have default interface implementations, so extending an interface is considered breaking). Though, those languages could likely use two interfaces.

Just highlighting it because it would be easy to accidentally do this in a breaking way.

(Similarly interface extenstion may be seen as a breaking change for testing, because people implement client interfaces and things, and may not have default implementations.)

Reply:
Absolutely. I think SDK implementations will have to be careful in implementing this. As much as possible we'd like to avoid a new major version.

Reply:
@Kavindu-Dodan has some good thoughts about this WRT the Go SDK.